src/NimBLEHIDDevice.cpp

```diff
@@ -128,8 +128,8 @@ void NimBLEHIDDevice::hidInfo(uint8_t country, uint8_t flags) {
  * @return pointer to new input report characteristic
  */
 NimBLECharacteristic* NimBLEHIDDevice::inputReport(uint8_t reportID) {
-	NimBLECharacteristic* inputReportCharacteristic = m_hidService->createCharacteristic((uint16_t) 0x2a4d, NIMBLE_PROPERTY::READ | NIMBLE_PROPERTY::NOTIFY);
-	NimBLEDescriptor* inputReportDescriptor = inputReportCharacteristic->createDescriptor((uint16_t) 0x2908);
```

Comment:
Admittedly I do not know HID devices well, (learned quite a bit since this was added, thank you) a question though, is the 0x2908 descriptor not needed for the input report? I'm assuming not but wanted to verify.

Reply:
Oops, too quick. Sorry I just noticed the changes below, please disregard.